pom.xml

```diff
@@ -21,7 +21,7 @@
     <groupId>ons.sdc.int.common</groupId>
     <artifactId>common</artifactId>
     <!-- change this to the version of ALL common libs to be used by this build -->
-    <version>1.0.29</version>
+    <version>1.0.31-SNAPSHOT</version>
```

Comment:
We can't merge dependency on a snapshot

Reply:
Done

Reply:
This has been done. As Rob is on holiday, I'll mark it as resolved.